torchio/transforms/preprocessing/intensity/projection.py

```diff
@@ -0,0 +1,158 @@
+import torch
+from torchio.data.image import ScalarImage
+from ....data.subject import Subject
+from ...intensity_transform import IntensityTransform
+from typing import Optional
+from math import ceil
+
+
+class Projection(IntensityTransform):
```

Comment:
What about `SlabProjection`? `Projection` doesn't feel very descriptive.

Reply:
I originally used `IntensityProjection` but thought it was redundant since it was an intensity transform. Happy to try something different.